test/_fixtures/error-payload.json

```diff
@@ -0,0 +1,12 @@
+{
+    "fullDisplayName": "node-test-commit-arm #3087",
+    "result": "FAILURE",
+    "changeSet": {
+        "items": [
+            {
+                "commitId": "7190bd23810f0880d95605bf6f785224a345d477"
+            }
+        ]
+    },
```

Comment:
We really just need the last commit, as the changes in previous commits would be included in the last commit because of git's parent refs. This is also visible in what Travis does to PRs, only the last commit included in the PR will get the "build state icon".